### DISCUSSION TITLE
Categorical Countplot

### DISCUSSION THREAD

**Discussion by szabzola** [hnolCol/instantclue#53] [Ideas]
It would be useful to have a vertical barchart representig the catergory sizes, as here:
![image](https://user-images.githubusercontent.com/61981001/157636804-21c05ccb-1fd5-47d6-9002-64a996377157.png)

Thanks for all the nice work.

**Comment by hnolCol** [comment]
Hi, thanks for your valuable feedback. I have implemented your request in a way that it looks like this (summing up the values across one "row" basically, hence the name total categorical counts - will think about this ;))
![image](https://user-images.githubusercontent.com/24191910/159294458-a30e1a8d-b990-47ad-8ee4-a9a113a50bab.png)
I will add options to add the category to the right barchart and also the total number will be displayed. 
It is implemented in the upcoming 0.11.2.

Best wishes

Hendrik
